src/test/test_proj_data_info.cxx

```diff
@@ -236,7 +236,7 @@ ProjDataInfoTests::test_generic_proj_data_info(ProjDataInfo& proj_data_info)
                             || !check(diff_view_num <= 1, "round-trip get_LOR then get_bin: view")
                             || !check(diff_axial_pos_num <= 1, "round-trip get_LOR then get_bin: axial_pos")
                             || !check(diff_tangential_pos_num <= 1, "round-trip get_LOR then get_bin: tangential_pos")
-                            || !check(diff_timing_pos_num == 0, "round-trip get_LOR then get_bin: timing_pos"))
+                            || !check(diff_timing_pos_num == 0 || diff_timing_pos_num ==  proj_data_info.get_max_tof_pos_num() , "round-trip get_LOR then get_bin: timing_pos"))
```

Comment:
why do we have this? If it's valid, it needs to be commented.

Reply:
this happens because the diff_timing_pos depends on view_num and when doing the roundtrip view_num goes from 0 to max and so diff_timing_pos=org_bin.timing_pos_num() + new_bin.timing_pos_num()
@KrisThielemans too long as a comment?

Reply:
I think that means that https://github.com/UCL/STIR/blob/605f950f56b6536a3dbe16658d9389219ebec1f2/src/test/test_proj_data_info.cxx#L215-L216 is incorrect. The swapping should have been detected by the check on views, so after that, the TOF positions should be close.

Reply:
so Are you saying diff_timing_pos_num=intabs(org_bin.timing_pos_num() - new_bin.timing_pos_num()) always?

Reply:
no, if view is 180 degrees, then the timing pos would have flipped sign in the previous case. Now, the timing pos would be `get_max_timing_pos_num() - org_timing_pos`, so I suppose that would mean
```
const int diff_timing_pos_num = intabs(org_bin.view_num() - new_bin.view_num()) < proj_data_info.get_num_views() - intabs(org_bin.view_num() - new_bin.view_num()) ?  
   intabs(org_bin.timing_pos_num() - new_bin.timing_pos_num()) : intabs(org_bin.timing_pos_num() - (get_max_timing_pos_num()  - new_bin.timing_pos_num())); 
```
it does make sense that this condition has to change w.r.t. the old case.

Obviously, it'd look nicer if we introduce a variable
```
const bool views_are_close = intabs(org_bin.view_num() - new_bin.view_num()) < proj_data_info.get_num_views() - intabs(org_bin.view_num() - new_bin.view_num())
```
or similar name, and use it for all checks.

Reply:
fixed! thanks